lib/templates/httpTemplate.ts

```diff
@@ -30,7 +30,9 @@ export class HttpTemplate {
 
   protected getHost(): string | undefined {
     const requestUrl = this.request.url;
-    return requestUrl ? url.parse(requestUrl).host : "management.azure.com";
+    return requestUrl
+      ? new URL(requestUrl, "https://management.azure.com").host
```

Comment:
Is this default domain when it's absent?

Reply:
yes